### DISCUSSION TITLE
İbax patlama yaparmi ?

### DISCUSSION THREAD

**Discussion by Brksay** [IBAX-io/go-ibax#135] [Bounty Hunter]
İbax ne kadar değerlenir avax seviyelerine gelebilir mi ?

**Comment by cihanaslan0** [comment]
> İbax ne kadar değerlenir avax seviyelerine gelebilir mi ?

Uzun vadede neden olmasın

**Comment by hemresenol** [comment]
Uzun vadede gelebilir ama şu an mümkün görünmüyor.
